devenv/config.ini

```diff
@@ -31,15 +31,16 @@ linux_arm64 = https://github.com/indygreg/python-build-standalone/releases/downl
 linux_arm64_sha256 = d37aef7bdf5c27f7d006918f7cedb31f4ba07c88f61baac4ffbe0bee6d4b5248
 
 [node]
-# upstream (https://nodejs.org/dist/) is not reliable enough so we've mirrored it to GCS
-darwin_x86_64 = https://storage.googleapis.com/sentry-dev-infra-assets/node/node-v22.11.0-darwin-x64.tar.xz
-darwin_x86_64_sha256 = ab28d1784625d151e3f608a9412a009118f376118ed842ae643f8c2efdfb0af6
-darwin_arm64 = https://storage.googleapis.com/sentry-dev-infra-assets/node/node-v22.11.0-darwin-arm64.tar.xz
-darwin_arm64_sha256 = c379a90c6aa605b74042a233ddcda4247b347ba5732007d280e44422cc8f9ecb
-linux_x86_64 = https://storage.googleapis.com/sentry-dev-infra-assets/node/node-v22.11.0-linux-x64.tar.xz
-linux_x86_64_sha256 = 83bf07dd343002a26211cf1fcd46a9d9534219aad42ee02847816940bf610a72
+# upstream (https://nodejs.org/dist/) is not reliable enough
+# ask someone in team-devinfra to upload for you
```

Comment:
@JonasBa i've mirrored it - added a comment here to just ask one of us to do it in the future